### PR TITLE
Add person class for retrieval of persons

### PR DESCRIPTION
**Pull request by kerrermanisNL**
Able to query, insert, update and delete `Person`s. I wasn't entirely sure how the `email` and `phone` fields should be handled, since they require an array and I couldn't find those. These work with the `BaseType` classes. However, that class is probably not meant to be used like that. Unfortunately I couldn't get anywhere trying to write my own, so I kept it at this. Let me know what you think about it, and I'll update it some.

I'm adding this, because I want to be able to add `Note`s to Pipedrive and I thought I might as well contribute to this repo. In my specific use case I need `Person`s before I can use `Note`s. You can expect a PR for that class as well.

### CODE REVIEW

File: pipedrive/models.py [hunk 1]
```diff
@@ -116,3 +116,16 @@ class ActivityType(Model):
...
+
+
+class Person(Model):
+    """
+    Model for Pipedrive persons.
+    """
+    name = StringType(required=True)
+    owner_id = PipedriveModelType(User, required=False)
+    org_id = PipedriveModelType(Organization, required=False)
+    email = BaseType(required=False)
```
Comment (kerrermanisNL) [inline pipedrive/models.py]: Perhaps I should wrap these in `ListType`?

**Comment by carlos-loggi** [comment]
Hello, Bono! First of all, sorry about the delay on answering to your PR. Also, thanks for taking interest in our library!

If you need an array-like field in a Schematics model, consider using ListType. I believe this is what you're looking for. More details in: http://schematics.readthedocs.io/en/latest/api/types.html#schematics.types.compound.ListType

**Comment by kerrermanisNL** [comment]
@carlos-loggi Hi Carlos, thanks for the pointer to the docs. I had already tried the `ListType`, but couldn't get it working with that. Turns out it returns a `List` containing `dict`s containing `string`s. So I've updated the model to represent this. I reckon it should be good now. Let me know if you would to see some things changed and I'll update them!

**Comment by carlos-loggi** [comment]
@kerrermanisNL I believe that it's the Pipedrive API that is returning a list of dicts when it should be returning a list of strings, right? One of the API's erratic behaviors that we noticed is that sometimes they returned a dict when they should return something else - mostly fields that should contain a numeric ID representing a foreign key contained the actual object and vice-versa. Lots of chaos and mayhem.

If that's the case here, I believe that the cleanest way to fit a square peg in a round hole and solve this is to create a custom type for this field - something like `EmailStringType` or `GoToHellPipedriveStringType`. Extend either `BaseType` or `StringType` and override the `to_native` and `to_primitive` methods to correctly handle what is received from/sent to Pipedrive. Then, you can create a `ListType` with it. Details here: http://schematics.readthedocs.io/en/latest/api/types.html

**Comment by kerrermanisNL** [comment]
Moved to #4

**Pull request closed without merging**